firefly/README.md

```diff
@@ -0,0 +1,31 @@
+# Firefly
+
+## Overview
+Firefly is a Cloud Asset Management solution that helps Cloud teams discover their cloud footprint (incl. AWS, GCP, Kubernetes, and Datadog), automatically transform resources to Infrastructure-as-Code, and detect drifts and policy violations to make sure their cloud is aligned with its desired state.
+Firefly helps teams make their Datadog assets immutable, versioned, scalable, and monitored by managing them as code in any Infrastructure-as-Code (IaC) tool of their choice.
+
+### Full cloud inventory
+Get a fully searchable inventory of all Datadog assets alongside other cloud assets, such as AWS, K8s, GCP, Okta, and more.
+
+### Automatic codification
+Automatically turn a single or multiple Datadog assets into code, including  Terraform, Pulumi, Cloudformation, and CDK.
+
+### Detect and fix drift
+Get real-time notifications on any discrepancy and push a fix directly to your repository as it occurs.
```

Comment:
Could you specify what kind of discrepancy would send a notification?

Reply:
The discrepancy is between the state of the cloud and what’s described in the IaC code, usually it’s referred to as “Drift”

Reply:
Gotcha, it would be great if you could add that context directly into the description here. Thanks!

Reply:
Get real-time notification of any discrepancy between your Infrastructure-as-Code and your actual state of cloud, and push a fix directly to your repository as drift occurs.